spec/features/audit/report/progress_spec.rb

```diff
@@ -1,43 +1,34 @@
 RSpec.feature "Reporting on audit progress" do
   scenario "Displaying the number of items included in the audit" do
-    given_i_have_been_allocated_audited_and_unaudited_content_items_belonging_to_my_organisation
+    given_i_am_an_auditor_belonging_to_an_organisation_with_content_items
+    when_i_visit_the_report_page
     then_i_see_all_content_items
-    given_that_select_a_document_type_that_does_not_apply_to_any_content_item
+    when_i_select_a_document_type_that_does_not_apply_to_any_content_item
     then_i_see_no_content_items
-    given_that_select_a_document_type_that_applies_to_all_content_items
-    then_i_see_all_content_items
+    when_i_select_a_document_type_that_does_apply_to_content_items
+    then_i_see_how_many_content_items_it_applies_to
```

Comment:
Again, don't shy away from being explicit. We're being very specific in the actual things we test, so we should describe those things at the high level:

> Given I ... belong to an organisation with **3** content items
> ...
> Then I see 3 content items